core/src/main/java/com/dtp/core/context/BaseNotifyCtx.java

```diff
@@ -16,12 +17,15 @@ public class BaseNotifyCtx {
 
     private ExecutorWrapper executorWrapper;
```

Comment:
The thread pool data is already snapshot when entering the alarm chain, so can we remove the ExecutorWrapper?